internal/access/access_test.go

```diff
@@ -94,14 +94,29 @@ func TestBasicGrant(t *testing.T) {
 }
 
 func TestUsersGroupGrant(t *testing.T) {
+	models.SkipSymmetricKey = true
+	t.Cleanup(func() {
+		models.SkipSymmetricKey = false
+	})
 	db := setupDB(t)
-	err := data.CreateIdentity(db, tom)
+
+	tom = &models.Identity{Name: "tom@infrahq.com"}
+	tomsGroup = &models.Group{Name: "tom's group"}
```

Comment:
These were defined at the package level previously. I think it's pretty risky to re-use these in tests because they are pointers that get updated by each test. It's also very little extra work to copy them into the test that uses them, so I did that for this one test I was changing. There are a few other tests that would probably benefit from the same cleanup.